templates/cook/cook_confirm_delete.html

```diff
@@ -0,0 +1,22 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+
+  <div class="container my-5">
+    <div class="card">
+      <div class="card-body">
+        <h5 class="card-title">Delete Cook</h5>
+        <p class="card-text">Are you sure you want to delete the cook: {{ cook }}?</p>
+        <form method="post">
+          {% csrf_token %}
+          {{ form|crispy }}
+          <div class="d-flex justify-content-end">
+            <a href="{% url 'management_system:cooks-list' %}" class="btn btn-secondary me-2">Cancel</a>
```

Comment:
me-2 class does not exist, i think you made a typo, probably mt-2 is correct

Reply:
fixed.